### ISSUE TITLE
Issues printing from ASP with "Kevin Leonard's Turnkey MVT with ASP and HASP"

### ISSUE THREAD

**Issue by cjar1950** [Fish-Git/hyperion#86]
I am having issues printing to a 1403 printer with _"[Kevin Leonard's Turnkey MVT with ASP and HASP](http://www.ibiblio.org/jmaynard/)"_ (specifically ASP).

With the [SDL Hyperion 4.0.0.8906-SDL-g1035eafe](http://www.softdevlabs.com/hyperion.html#prebuilt#8906) windows pre-built, I am getting _"Buffer Load Check"_ errors from ASP and "`LOADCK`" errors from Hercules.

I have tracked back as far as I can, and have the following results:
&nbsp;

Version | Build Date | ASP Results | Hercules message
--------|------------|-------------|------------------
3.10.0.0 | 2014-22-01 | OK |
3.12 | 2015-11-26 | OK |
hercules464-rc0-4.0.0 | 2016-12-16 | OK |
4.00.0.8243 | 2015-10-06 |  OK |
4.00.0.8473 | 2016-04-23 | OK |
4.0.0.8650-SDL | 2016-10-14 | OK |
4.0.0.8701-SDL | 2016-12-23  |  OK |
4.0.0.8745-SDL | 2017-01-31 | _Equipment Check_ |
4.0.0.8775-SDL | 2017-02-21 | _Equipment Check_ |
4.0.0.8845-SDL | 2017-06-06 | _Buffer Load Check_ | `HHC01314I 0:000E CHAN: sense CCK`
4.0.0.8873-SDL | 2017-07-08 | _Buffer Load Check_ | `HHC01314I 0:000E CHAN: sense LOADCK`
4.0.0.8906-SDL | 2017-11-07 | _Buffer Load Check_ | `HHC01314I 0:000E CHAN: sense LOADCK`

&nbsp;
All of these tests used the same DASD, and (as far as possible) the same config file. The config entry for the printer is:

```
  000E   1403   "C:/Users/Public/Hercules/MVT/asp/prtx/prt00e.txt"   crlf
```

HASP actually works without any problems.

Chris.

**Comment by Fish-Git** [comment]
@cjar1950:

I am going to need to see a CCW trace (`t+00e`).

**Comment by cjar1950** [comment]
[Hercules Logfile.256.txt](https://github.com/Fish-Git/hyperion/files/1875247/Hercules.Logfile.256.txt)

**Comment by Fish-Git** [comment]
Got it! Thank you. I will analyze it and get back to you.

**Comment by Fish-Git** [comment]
Analysis complete: this appears to be a bug in the printer driver.

I will try to get it fixed as soon as I can.

**Comment by Fish-Git** [comment]
Fixed by bff81c3896ee08c20d61b0690315803c5133e3f9.
